app/controllers/pulls_controller.rb

```diff
@@ -85,6 +85,8 @@ def show
 
     @diff_type = get_pull_diff_type
 
+    @conflicting_files = @pull.repository.conflicting_files(@pull.commit_base, @pull.commit_head)
```

Comment:
Can you encapsulate this in a method `conflicting_files` in the `Pull` class? Also this new method should return an empty array if `pull.merge_status != 'conflicts_detected'`. That way, we will avoid Git call if it's unnecessary.

If you cache the `conflicting_files` result in the `Pull` class, you also won't need to define a global variable here. You can use it in the view directly. See `Pull#revisions_ids` for an example of caching in the `Pull` class.